packages/hint/src/lib/utils/fs/read-file.ts

```diff
@@ -3,6 +3,6 @@ import { readFileSync } from 'fs';
 import stripBom = require('strip-bom'); // `require` used because `strip-bom` exports a function
 
 /** Convenience wrapper for synchronously reading file contents. */
-export default (filePath: string): string => {
+export const readFile = (filePath: string): string => {
```

Comment:
Why did you remove the default? Is it to make it easier to refactor?

Reply:
This was for 2 reason:
1. It is easy to export.
1. To use the same method name everywhere, I mean, if we are exposing the method: `readFile`, it would be great that we are using also `readFile` everywhere, and not `readFile` in one place `readFileSync` in another, etc. I don't know if I'm explaining myself hehehe.

Reply:
> if we are exposing the method: readFile, it would be great that we are using also readFile everywhere, and not readFile in one place readFileSync in another, etc. I don't know if I'm explaining myself hehehe.

This is just an example.